<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUIAdmin</class>
 <widget class="QWidget" name="GUIAdmin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUIAdmin</string>
  </property>
  <widget class="QListWidget" name="repoList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>521</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QPushButton" name="updateButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>421</width>
     <height>197</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="titleLabel">
      <property name="text">
       <string>Title</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="presenterLabel">
      <property name="text">
       <string>Presenter</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="presenterEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="likesLabel">
      <property name="text">
       <string>Likes</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="likesEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="durationLabel">
      <property name="text">
       <string>Duration</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="durationEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="linkLabel">
      <property name="text">
       <string>Link</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="linkEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="viewsLabel">
      <property name="text">
       <string>Views</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="viewsEdit"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="filterByViewsLabel">
      <property name="text">
       <string>Filter by views</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="filterByViewsEdit"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="titleEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="filterButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>360</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Filter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="displayButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>400</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Display</string>
   </property>
  </widget>
  <widget class="QComboBox" name="detailComboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>521</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Detailed</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Short version</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="undoButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="redoButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>440</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
